react-graph/src/LineLegend.tsx

```diff
@@ -40,14 +40,14 @@ export interface IProps extends ILegendRequiredProps {
 const nonTextualWidth = 45;
 const textFont = "Segoe UI";
 function LineLegend(props: IProps) {
+  const fontFamily = `-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol"`
```

Comment:
This should be moved outside of the react component. Otherwise, every render this is reassigned.